### ISSUE TITLE
Create the metadata service screen

### ISSUE THREAD

**Issue by cherrera2001** [hashmapinc/Tempus#461]
Child of #462
Depends on #460

The data model service screen will be the UI that is displayed when the user click on either the sidebar or the icon in the dashboard.

For this Task we are simply implementing an empty screen. We will build upon this later for the catalog, but this is simply so we can implement the data object creation flow.

User Type:

Tenant Administrator

User Entry Point:

The user will enter this screen from either the sidebar or by clicking the Metadata icon on the dashboard.

Out of scope:

For this card we are not implementing viewing of the metadata sources, or listing them. We are simply getting to a page that is going to allow us to create a new metadata source. To view them or delete them, this will be a manual task in the database.

Design:

The data model service screen will be the UI that displays the metadata sources that are currently in tempus. It will display a plain grey background with the single orange add button in the lower right as normal on every other page:
